app/views/pages/home.html.erb

```diff
@@ -0,0 +1,10 @@
+<div class="hero">
+	<div class="hero-inner">
```

Comment:
We're using spaces rather than tabs everywhere else, might be good to keep it consistent?